code/modules/hydroponics/grown/towercap.dm

```diff
@@ -119,6 +119,15 @@
 /obj/structure/bonfire/dense
 	density = TRUE
 
+/obj/structure/bonfire/lit //haha empty define
```

Comment:
technically this is redundant because you're already defining a child type of this

Reply:
Is it better to have a shell type explicitly defined, or to imply it exists?

Reply:
that's a good question honestly, I'm curious what other people think cause I don't know our standards for it (or if we have them). For code clarity sake, it's probably better to explicitly define it?

Reply:
We usually prefer to explicitly define it.